cmake/JuliaCCompiler.cmake

```diff
@@ -0,0 +1,84 @@
+#
+
+# libc++ is standard on OS X 10.9, but not for earlier releases
+jl_option(USE_LIBCPP "Use libc++" Off)
+# assume we don't have LIBSSP support in our compiler,
+# will enable later if likely true
+set(HAVE_SSP Off)
+
+jl_set_option(USEICC Off)
+jl_set_option(USEGCC Off)
+jl_set_option(USECLANG Off)
+jl_set_option(USEMSVC Off)
```

Comment:
I'm proposing saving some typing by dropping the `USE` and the logic to set these once they aren't being sent to existing makefiles, since cmake already has a `MSVC` auto-variable.